docs/users-guide/09-multi-series-charting.md

```diff
@@ -7,73 +7,79 @@ Data in isolation is rarely all that useful. One of the best ways to add context
 - New users per day vs. returning users per day.
 - Orders per day from a few different product lines.
 
-### In Metabase there are two main ways to get data side-by-side.
+### Two ways to display data side-by-side
 
-1. Combining two existing saved questions that share a common dimension (like time) on a dashboard
+1. [**Use the notebook editor to ask a question that involves multiple dimensions**](#using-the-notebook-editor-to-create-a-multi-series-visualization) (or in SQL, if you’re fancy). Example: the count of users by region over time.
 
-e.g. Let me see revenue over time and cost over time together.
+2. [**Combine two saved questions**](#combining-two-saved-questions) that share a common dimension (like time) on a dashboard. For example, let me see revenue over time and cost over time together.
 
-2. Asking a question that involves multiple dimensions in the query builder (or in SQL if you’re fancy).
+### Using the notebook editor to create a multi-series visualization
```

Comment:
Ah, so I'm seeing here that there's some terminology confusion: "the notebook editor" is what we sometimes call the Custom Question mode. What you're describing below is the Simple Question mode.

The way you'd do this same thing in the Custom Question mode is to add two columns in the group-by section of the Summarize block.